ospf6d/ospf6_asbr.c

```diff
@@ -1578,6 +1589,9 @@ DEFUN (ospf6_redistribute,
 	if (!red)
 		return CMD_SUCCESS;
 
+	red->dmetric.type = def_metric_type;
```

Comment:
Let's move it to `ospf6_redist_add` to reduce unnecessary code duplication. Like it's done in ospfd.

Reply:
@idryzhov fixed.